src/pz390.java

```diff
@@ -390,7 +390,8 @@
      * 2021-10-29 Issue 305. Fix XHEXI logic so that it matches ASSIST documentation.
      *                       NB: RPI 878 added ASCII support for XDECI but not XHEXI.
      *                           Not added here either.
-     * 2022-02-12 Issue 195. For all privileged instructions issue the proper exception according PoP
+	 * 2022-02-24 DSH ISSUE 330 add support for MGRK, MSGRKC, MSRKC, MSC
+   * 2022-02-12 Issue 195. For all privileged instructions issue the proper exception according PoP
 	 *                       Being S0C2 for privileged-operation exception when executed in problem state
 	 *                       Other exceptions as defined in PoP
 	 *                       And implement as a no-op if all tests are passed
```

Comment:
"ISSUE 330 add ..." should be "Issue 300 add ...". Correct?

A nit, but the 2022-02-24 comment should be below the 20220212 comment.